### PR TITLE
amixer, fatlabel, i7z, vigr, vpnc: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/linux/amixer.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# amixer
+
+> Mikser dla sterownika ALSA kart dźwiękowych.
```
Comment (kbdharun) [inline pages.pl/linux/amixer.md:3]:
```suggestion
> Mikser dla sterownika ALSA kart dźwiękowych.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. With the above suggestion applied this PR is GTG.

File: pages.pl/linux/fatlabel.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> Ustawia lub pobiera informację o nazwie partycji FAT32.
+> Więcej informacji: <https://manned.org/fatlabel>.
+
+- Pobranie informacji o nazwie partycji FAT32:
+
+`fatlabel {{/dev/sda1}}`
+
+- Ustawienie nazwy partycji FAT32:
+
+`fatlabel {{/dev/sdc3}} "{{new_label}}"`
```
Comment (kbdharun) [inline pages.pl/linux/fatlabel.md:12]:
```suggestion
`fatlabel {{/dev/sdc3}} "{{nowa_etykieta}}"`
```
Checked this with Google Translation. Would this work here?

**Review by kbdharun — COMMENTED** [review]
Seems like you have created a separate PR (#9416) for `i7z` and `vigr` can you remove it here?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**